keg_login/tests/test_middleware.py

```diff
@@ -0,0 +1,17 @@
+import keg_login.middleware as middleware
+
+
+class TestLockedOutMiddleware(object):
+    def test_excluded_routes(self):
+        cls = middleware.LockedOutMiddleware
```

Comment:
NIT: I'd avoid using `cls` as an assigned variable name in the same way I'd avoid using `self`.